src/main/kotlin/app/revanced/patches/music/ad/video/fingerprints/ShowMusicVideoAdsFingerprint.kt

```diff
@@ -0,0 +1,21 @@
+package app.revanced.patches.music.ad.video.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.annotation.DirectPatternScanMethod
+import app.revanced.patcher.fingerprint.method.annotation.MatchingMethod
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patches.music.ad.video.annotations.MusicVideoAdsCompatibility
+import org.jf.dexlib2.AccessFlags
+
+@Name("show-video-ads-method-fingerprint")
+@MatchingMethod(
+    definingClass = "zai"
+)
+@DirectPatternScanMethod
+@MusicVideoAdsCompatibility
+@Version("0.0.1")
+object ShowMusicVideoAdsFingerprint : MethodFingerprint(
+    "V", AccessFlags.PUBLIC or AccessFlags.FINAL, listOf("Z"), null
```

Comment:
This fingerprint looks like it will not match to the desired method.

Reply:
Fixed this too